### ISSUE TITLE
Can't serve two apps at the same time

### ISSUE THREAD

**Issue by brandyscarney** [driftyco/ionic-app-scripts#421]
I get the following error when trying to serve a second app while one is already served:

```
brandyscarney:~/Documents/2.0-sdk-help/myApp
$ ionic serve

> ionic-hello-world@ ionic:serve /Users/brandyscarney/Documents/2.0-sdk-help/myApp
> ionic-app-scripts serve

[15:30:17]  ionic-app-scripts 0.0.45
[15:30:17]  watch started ...
[15:30:17]  build dev started ...
[15:30:17]  clean started ...
[15:30:17]  clean finished in 2 ms
[15:30:17]  copy started ...
[15:30:17]  transpile started ...
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE 0.0.0.0:53703
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at Server._listen2 (net.js:1234:14)
    at listen (net.js:1270:10)
    at net.js:1379:9
    at doNTCallback3 (node.js:461:9)
    at process._tickCallback (node.js:367:17)
    at Function.Module.runMain (module.js:459:11)
    at startup (node.js:136:18)
    at node.js:972:3

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/brandyscarney/.nvm/versions/node/v5.0.0/bin/node" "/Users/brandyscarney/.nvm/versions/node/v5.0.0/bin/npm" "run" "ionic:serve" "--"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
npm ERR! ionic-hello-world@ ionic:serve: `ionic-app-scripts serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ionic-hello-world@ ionic:serve script 'ionic-app-scripts serve'.
npm ERR! This is most likely a problem with the ionic-hello-world package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ionic-app-scripts serve
npm ERR! You can get their info via:
npm ERR!     npm owner ls ionic-hello-world
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/brandyscarney/Documents/2.0-sdk-help/myApp/npm-debug.log
There was an error serving your Ionic application: There was an error with the spawned command: serve
```

and a similar error when passing the port in. Discussed this with @jthoms1

**Comment by jthoms1** [comment]
It appears that the Notification port is set to use port 53703. This is never passed from the CLI. So we need to either update the CLI to start passing this port or app-scripts should identify a different port if this one is used.

**Comment by rapropos** [comment]
#383 was closed and moved to [cli #1678](https://github.com/driftyco/ionic-cli/issues/1678).

**Comment by jthoms1** [comment]
Sorry @brandyscarney this is a dupe that I forgot was covered in a cli ticket.  I will ensure that I move over the new details on what we learned about the issue.
